core/process.py

```diff
@@ -183,6 +186,7 @@ def __init__(
             mem,  #type: state.Mem
             tracer,  # type: Optional[dev.Tracer]
             waiter,  # type: Optional[Waiter]
+            mutable_opts,  #type: MutableOpts
```

Comment:
If we just reading this bit, please use `self.exec_opts.noclobber()`

There is some metaprogramming that allows that simpler interface

self.mutable_opts is for mutating options

Reply:
replaced with exec_opts. I tried to use bool instead and call with exec_opts.noclobber() but it return always False, so I guess these are lately set.